src/Router/Adapter.php

```diff
@@ -159,10 +151,10 @@ function () use ($e, $dataModel, $message) {
     private function handleException(Message $message, \Throwable $e): void
     {
         $onCompleted = function () use ($e, $message) {
-            echo "Message {$message->getRoutingKey()} has been nack".PHP_EOL;
-            echo "Reason: {$e->getMessage()}".PHP_EOL;
-            echo "This occurs in {$e->getFile()} @line {$e->getLine()}".PHP_EOL;
-            $this->logger->error($e->getMessage(), [
+            $this->logger->error("Message {$message->getRoutingKey()} has been nack", [
+                'reason' => $e->getMessage(),
```

Comment:
IMO `reason|file|line` it's not needed because we are capturing the whole exception.

Have you test the render in Sentry and stdout ?